documentation/docs/01-routing.md

```diff
@@ -48,27 +48,27 @@ A file or directory can have multiple dynamic parts, like `[id]-[category].svelt
 Endpoints are modules written in `.js` (or `.ts`) files that export functions corresponding to HTTP methods. For example, our hypothetical blog page, `/blog/cool-article`, might request data from `/blog/cool-article.json`, which could be represented by a `src/routes/blog/[slug].json.js` endpoint:
 
 ```ts
-type Request<Context = any> = {
-	host: string;
+type Request<Locals = Record<string, any>> = {
 	method: 'GET';
-	headers: Record<string, string>;
+	host: string;
+	headers: Headers;
 	path: string;
-	params: Record<string, string | string[]>;
+	params: Record<string, string>;
 	query: URLSearchParams;
 	rawBody: string | Uint8Array;
 	body: string | Uint8Array | JSONValue;
-	locals: Record<string, any>; // see below
+	locals: Locals; // see hooks handle
```

Comment:
"see hooks handle" isn't very clear. see it for info about what? maybe something more like "populated by hooks handle"

Reply:
That might be better 👍